src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/CreditorsAfterOneYearServiceImpl.java

```diff
@@ -54,6 +66,44 @@ public CreditorsAfterOneYear getCreditorsAfterOneYear(String transactionId,
         return creditorsAfterOneYear;
     }
 
+    @Override
+    public List<ValidationError> submitCreditorsAfterOneYear(String transactionId,
+            String companyAccountsId, CreditorsAfterOneYear creditorsAfterOneYear,
+            String companyNumber) throws ServiceException {
```

Comment:
Is `companyNumber` used at all?

Reply:
removed company number as doesn't seem to be being used